config/sidekiq.yml

```
@@ -3,11 +3,11 @@
 # sidekiq -C config.yml
 ---
 :verbose: <%= ENV.fetch('SIDEKIQ_VERBOSE', false) %>
```

Similar question to the one I asked for Caesar – does this need `.to_bool`?

---

no - in ruby everything is `truthy` or `falsey`, e.g 
```
irb
> ENV.fetch('SIDEKIQ_VERBOSE', false)
 => false 
2.5.3 :008 > puts "I am truthy" if ENV.fetch('SIDEKIQ_VERBOSE', false)
 => nil 
2.5.3 :002 > ENV['SIDEKIQ_VERBOSE'] = 'true'
 => "true" 
2.5.3 :003 > ENV.fetch('SIDEKIQ_VERBOSE', false)
 => "true" 
2.5.3 :005 > !!ENV.fetch('SIDEKIQ_VERBOSE', false)
 => true 
2.5.3 :006 > puts "I am truthy" if ENV.fetch('SIDEKIQ_VERBOSE', false)
I am truthy
 => nil 
```

---

and sidekiq cli will parse this into the correct value - that said it will make it conformant to the default.
```suggestion
:verbose: <%= !!ENV.fetch('SIDEKIQ_VERBOSE', false) %>
```